api/KeyboardEvent.json

```diff
@@ -901,10 +901,10 @@
                 "version_added": "35"
               },
               "safari": {
-                "version_added": null
+                "version_added": false
```

Comment:
This "OS" modifier isn't in the FIXME, but it's also not one of the supported strings in the source.